### PR TITLE
Homepage template requested fixes

### PR DESCRIPTION
**Pull request by mtuchi**
## Description

Fixed merge conflicts, from requested changes, Also added editor config file 

Fixes #37 

## Type of change

Please delete options that are not relevant.

- [x] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## Screenshots
home

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -30,16 +34,17 @@ create user takwimu with password takwimu;
...
 Import the data into the new database (will overwrite some tables created by Django, but that's ok).
 ```
 cat takwimu/sql/*.sql | psql -U takwimu -W takwimu
 ```
 
+Run migrations to keep Django happy:
+```
+python manage.py migrate
+```
+
```
Comment (DavidLemayian) [inline README.md]: Migrate should always come before loading data from SQL. Let's return this.

File: takwimu/settings.py [hunk 2]
```diff
@@ -56,6 +56,10 @@
 WAZIMAP = HURUMAP
 
 
+# -------------------------------------------------------------------------------------
+# Database Configs
+# -------------------------------------------------------------------------------------
+
 # -------------------------------------------------------------------------------------
 # Database Configs
 # -------------------------------------------------------------------------------------
```
Comment (DavidLemayian) [inline takwimu/settings.py]: ThThis shouldn't be here.

File: takwimu/templates/login.html [hunk 3]
```diff
@@ -0,0 +1,67 @@
...
+    						<input type="password" class="form-control" id="exampleInputPassword1" placeholder="Password">
+    					</div>
+    					<button type="submit" class="btn btn-primary">Login</button>
+    				</form>
+    			</div>
+    		</div>
+	</div>
+</div>
+    </div>
+{% endblock %}
```
Comment (DavidLemayian) [inline takwimu/templates/login.html]: Please fix indentation.

File: README.md (hunk 1 above)
Comment (mtuchi) [reply README.md]: I think @phillipahereza mention that the other command should start first, 
@phillipahereza what was the reason again for starting with 
```
cat takwimu/sql/*.sql | psql -U takwimu -W takwimu
```
Then 
```
python manage.py migrate
```

Comment (phillipahereza) [reply README.md]: @mtuchi run `python manage.py migrate` first, it saves you a lot of headache with the loading the geographies mostly

Comment (mtuchi) [reply README.md]: @phillipahereza :+1:

**Comment by mtuchi** [comment]
@DavidLemayian  looks like we have a green light on this one.

**Pull request merged**